trackpy/framewise_data.py

```diff
@@ -8,8 +8,8 @@
 
 from .utils import print_update
 
-__all__ = ['FramewiseData', 'PandasHDFStore', 'PandasHDFStoreBig',
-           'PandasHDFStoreSingleNode']
+__all__ = ['PandasHDFStore', 'PandasHDFStoreBig', 'PandasHDFStoreSingleNode',
```

Comment:
I made the same change in two different PRs by mistake. So, this change is not consequential.